### ISSUE TITLE
Add SSEM to PEcAn

### ISSUE THREAD

**Issue by ashiklom** [PecanProject/pecan#827]
Super Simple Ecosystem Model -- could be fantastic for testing, and should take like a day to implement in any language (in fact, Mike already has it implemented for his hands-on activities).

**Comment by ashiklom** [comment]
https://github.com/EcoForecast/EF_Activities/blob/master/Exercise_10_ParticleFilter.Rmd

**Comment by mdietze** [comment]
In addition, the BayesianTools package has a Very Simple Ecosystem Model that's even simpler than SSEM and is written in Rcpp so is also faster.  Agree that it would be useful to have a few models that are so simple that they approach being "null" models

**Comment by dlebauer** [comment]
I think we will tackle this as an ice breaker activity on day 1 of the retreat

**Comment by dlebauer** [comment]
@mdietze do you recommend we start w/ ssem or vsem? 

Should we just create an Rscript and handle it [like we do in BioCro](https://github.com/PecanProject/pecan/blob/develop/models/biocro/inst/biocro.Rscript)?

**Comment by mdietze** [comment]
I don't think that tackling this as an 'ice breaker' activity is a good idea as it could easily take a full day. I'd strongly recommend just diving directly into looking at the existing FATES coupler and/or looking at the documentation for coupling and a few of the simpler couplers (e.g. SIPNET, DALEC, JULES)
